packages/web-components/src/components/va-privacy-agreement/va-privacy-agreement.scss

```diff
@@ -23,4 +23,8 @@ a {
 .privacy-policy-icon {
   margin-left: 4px;
   vertical-align: text-top;
+}
+
+a:focus {
+  outline: 2px solid var(--vads-color-action-focus-on-light);
```

Comment:
Does importing (modifying it if necessary) the mixin not work?

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/mixins/focusable.css

For example: 
https://github.com/department-of-veterans-affairs/component-library/blob/c021b5f55dd524926eabe7f78d3014022ff386a3/packages/web-components/src/components/va-minimal-footer/va-minimal-footer.scss#L1

Reply:
You're right, I missed that. I've pushed an update now that imports and modifies the mixin as needed. Thanks for catching that!